desktop/build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+import org.jetbrains.compose.compose
+import org.jetbrains.compose.desktop.application.dsl.TargetFormat
+
+plugins {
+    kotlin("multiplatform")
+    id("org.jetbrains.compose") version "1.0.0-beta5"
+}
+
+group = "me.lierxiao"
```

Comment:
Is this correct? Not sure what this group is in reference to, I think maybe it should be `com.godaddy.android`?

Reply:
Sorry  i forgot to change the local package name 😂 ， i will check the issue  u mentioned above

Reply:
The last problem is confused I'm not sure if the computer and phone display mode is not the same 😂， Do you have any ideas？